CMakeLists.txt

```diff
@@ -67,10 +85,21 @@ add_library(gtest_170_lib ${GTEST_DIR}/src/gtest-all.cc)
 set_target_properties(gtest_170_lib PROPERTIES COMPILE_DEFINITIONS "GTEST_HAS_RTTI=0")
 enable_testing(true)
 
-include_directories(test)
-include_directories(${PROJECT_SRC})
-file(GLOB TEST_SRC_FILES "test/*.cpp")
 
-add_executable(UnitTestRunner 3rdparty/test_main.cpp ${TEST_SRC_FILES} )
```

Comment:
Please pull and merge the latest from master.

Today we updated this repo with important CMake changes